### PR TITLE
feat: add sepolia to allow chain for dev-unstable

### PR DESCRIPTION
**Pull request by 3benbox**
Add Sepolia chain id to the allowed list for dev-unstable.

### CODE REVIEW

File: packages/core/src/initialization/anchoring.ts [hunk 1]
```diff
@@ -18,7 +18,7 @@ export const DEFAULT_ANCHOR_SERVICE_URLS = {
 const SUPPORTED_CHAINS_BY_NETWORK = {
   [Networks.MAINNET]: ['eip155:1'], // Ethereum mainnet
   [Networks.TESTNET_CLAY]: ['eip155:3', 'eip155:4', 'eip155:100'], // Ethereum Ropsten, Rinkeby, Gnosis Chain
-  [Networks.DEV_UNSTABLE]: ['eip155:3', 'eip155:4', 'eip155:5'], // Ethereum Ropsten, Rinkeby, Goerli
+  [Networks.DEV_UNSTABLE]: ['eip155:3', 'eip155:4', 'eip155:5', 'eip155:11155111'], // Ethereum Ropsten, Rinkeby, Goerli, Sepolia
```
Comment (stbrody) [inline packages/core/src/initialization/anchoring.ts:21]: lol what an absurd chainid

**Review by stbrody — CHANGES_REQUESTED** [review]
I don't think this change is sufficient.  There are several other places in our codebase that reference chainids.  Just grep for "eip155:5" to see all the places we use the Goerli chainid today.  I assume most if not all of those places will need to be updated

File: packages/anchor-utils/src/constants.ts [hunk 2]
```diff
@@ -4,13 +4,15 @@ export type SupportedNetwork =
   | 'eip155:5' // goerli
   | 'eip155:100' // gnosis
   | 'eip155:1337' // ganache
+  | 'eip155:11155111' // sepolia
 
 export const BLOCK_THRESHHOLDS: Record<SupportedNetwork, number> = {
```
Comment (stbrody) [inline packages/anchor-utils/src/constants.ts:9]: looks like this constant isn't actually used anywhere, let's just delete it.

Comment (stbrody) [reply packages/anchor-utils/src/constants.ts:9]: note there's a similar constant in ethereum-anchor-validator.ts that *is* used however, and still need to be updated.

Comment (stbrody) [reply packages/anchor-utils/src/constants.ts:9]: ping

Comment (3benbox) [reply packages/anchor-utils/src/constants.ts:9]: The type is used on line 9 above, no?
Can that be removed too?

Comment (stbrody) [reply packages/anchor-utils/src/constants.ts:9]: sorry, this comment is *on* line 9, so I'm not sure what you're referring to.  I'm saying the `BLOCK_THRESHHOLDS` constant in this file is defined but never used.

Also there's a `BLOCK_THRESHHOLDS` constant in `ethereum-anchor-validator.ts` that *is* used and I think still needs to be updated.

**Review by stbrody — APPROVED** [review]
lgtm

**Pull request merged**